samza-test/src/main/java/org/apache/samza/test/integration/LocalApplicationRunnerMain.java

```diff
@@ -53,14 +53,14 @@ public static void main(String[] args) throws Exception {
 
     try {
       LOGGER.info("Launching stream application: {} to run.", app);
-      runner.run(buildExternalContext(config).orElse(null));
+      runner.run(buildExternalContext().orElse(null));
       runner.waitForFinish();
     } catch (Exception e) {
       LOGGER.error("Exception occurred when running application: {}.", app, e);
     }
   }
 
-  private static Optional<ExternalContext> buildExternalContext(Config config) {
+  private static Optional<ExternalContext> buildExternalContext() {
```

Comment:
The intention to have the "unused" `config` was so that a custom fork could use the `config` if necessary (see comment inside `buildExternalContext`. Could we please keep the `config`?

Reply:
Added back.